src/tracing/perfetto/BUILD.gn

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+# Configuration for using perfetto tracing macros exclusively.
+config("tracing") {
+  include_dirs = [ "include" ]
+
+  configs = [ "${chip_root}/third_party/perfetto:sdk_config" ]
+}
+
+static_library("perfetto") {
+  sources = [
+    # NOT a direct dependency, but available through the ":tracing" config:
+    #   "include/matter/tracing/macros_impl.h",
```

Comment:
This is not OK, please list all headers.

You can use "public" and "sources" to differentiate public and private headers.

Reply:
Updated to have a separate source set for macros_impl.h.